server/routes/visit.ts

```diff
@@ -99,6 +104,22 @@ export default function routes(
         res.locals.appInsightsOperationId,
       )
 
+      if (config.apis.notifications.enabled) {
+        try {
+          const phoneNumber = visit.visitContact.telephone.replace(/\s/g, '')
+
+          await notificationsService.sendCancellationSms({
+            phoneNumber,
+            visit: visit.startTimestamp,
+            prisonName: 'Hewell (HMP)',
+            prisonPhoneNumber: '01234443225',
+          })
+          logger.info(`Cancellation SMS sent to ${visit.visitContact.telephone}`)
```

Comment:
Here (and in the `error` log just below), I'd log the visit `reference` rather than the phone number to avoid personal data getting into logs. E.g. like what's done here: https://github.com/ministryofjustice/book-a-prison-visit-staff-ui/blob/62b63a6c4a72be7cbe8a62f37114daab8e838c65/server/routes/bookAVisit.ts#L494

Reply:
What @tpmcgowan said 🙂

Reply:
Great thanks both!